core/include/detray/tracks/bound_track_parameters.hpp

```diff
@@ -151,6 +151,32 @@ struct bound_track_parameters {
         matrix_operator().element(m_vector, e_bound_qoverp, 0) = qop;
     }
 
+    DETRAY_HOST_DEVICE
```

Comment:
This does not belong into the PR, I guess?

Reply:
Right.